components/Toast/toast.css

```diff
@@ -0,0 +1,32 @@
+.success-Toast {
+    background-color: green;
+    margin-left: 2rem;
```

Comment:
Here you used margin right and bottom separately you can use all at once

Reply:
okkk thansk for your suggestion